src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -701,9 +700,7 @@ try
         wheelDelta = HIWORD(wParam);
 
         // If it's a *WHEEL event, it's in screen coordinates, not window (?!)
-        auto coordsToTransform = cursorPosition.to_win32_point();
-        ScreenToClient(_hwnd.get(), &coordsToTransform);
-        cursorPosition = til::point{ coordsToTransform };
+        ScreenToClient(_hwnd.get(), cursorPosition.as_win32_point());

```

Comment:
I am deeply uncomfortable with this, but I know why you did it and why it is safe. Still, is it worth making people uncomfortable to save a temporary and two copies?

Reply:
Some legacy code makes it cumbersome to not to use `as_win32_point` and friends and so I believe these methods need to exist anyways. I think it's better to clean this up later, as it helped me write this PR faster. (I can convert the code `HwndTerminal.cpp` if you'd like but there's a lot more uses of this function and others unfortunately.)